### DISCUSSION TITLE
Change default font-display in @next/font to swap

### DISCUSSION THREAD

**Discussion by NiedziolkaMichal** [vercel/next.js#43178] [Ideas]
### Describe the feature you'd like to request

I believe default [font-display](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) descriptor defined in [@next/font/utils.ts](https://github.com/vercel/next.js/blob/canary/packages/font/src/google/utils.ts) should be changed from `optional` to `swap`. This is default value for all [https://fonts.google.com/](https://fonts.google.com/) fonts. [Article in smashing magazine](https://www.smashingmagazine.com/2021/05/reduce-font-loading-impact-css-descriptors/) recommends using either block or swap for critical fonts, while optional for `fonts that have already been downloaded`. [Article in developer.google.com](https://developer.chrome.com/blog/font-display/) states that `optional` should be used `when the downloading font is more of a “nice to have” but not critical to the experience`.

On latest chrome with disabled cache, using following font ALWAYS uses fallback instead:
```js
import {Dancing_Script} from '@next/font/google';

const font = Dancing_Script({ subsets: ['latin'] })
```

**Comment by jonathanwilke** [comment]
Found the same problem with my project. I agree, swap should be the default here.

**Comment by NiedziolkaMichal** [comment]
[Next.JS 13.2](https://nextjs.org/blog/next-13-2#other-improvements) changed the default value of `font-display` to `swap`. Thanks!
